### PR TITLE
developed motor movement functionality

### PR DESCRIPTION
**Pull request by PeterTWinkler**
This feature enables the robot to move along one chosen axis based on distance or a given number of steps.

### CODE REVIEW

**Comment by PaulBueckhard** [comment]
Changes were made to `stepper_motor_test.py`, but to my knowledge the file is unused. Commit these changes or delete the file?

File: stepper_motor_test.py
Comment (PaulBueckhard) [inline stepper_motor_test.py:1]: Deleting file, since it is unused and will conflict with pytests

**Review by PaulBueckhard — APPROVED** [review]
Changes seem to be fine and do not show any conflicts with main.

**Comment by koenidv** [comment]
Nice to finally see this working and merged haha

**Pull request merged**